src/linear_algebra/dual.lean

```diff
@@ -563,10 +577,6 @@ open_locale classical
 open finite_dimensional
 
 variables {V₁ : Type*} [add_comm_group V₁] [module K V₁]
-
-instance [H : finite_dimensional K V] : finite_dimensional K (module.dual K V) :=
-linear_equiv.finite_dimensional (basis.of_vector_space K V).to_dual_equiv
```

Comment:
I think we might still want this instance; or does `apply_instance` now find it?

Reply:
`apply_instance` finds it here but it doesn't work in [FinVect.FinVect_dual](https://leanprover-community.github.io/mathlib_docs/algebra/category/FinVect.html#FinVect.FinVect_dual), I'm not sure why.